luceda/python-lint-annotate
# use micromamba

- only python3 files that are added in PR are checked
- bash io sh
- only checksuite
- remove comment
- removed comment
- echo base commit, remove quotes
- pr
- remove quotes
- -l removed
- more like og flake8-your-pr
- test base commit
- undo
- forgot to run main
- apk
- pip install jq git curl
- conda install git, jq and curl
- debug info
- test
- sudo aptget install jq curl
- GITHUB_EVENT_PATH check
- test
- Revert "test"
- testing
- echo event.json
- clean up
- further clean up
- loop over files
- update flake8 version
- testing folder io file
- don't exit on crash
- clean up
- mypy problemmatcher
- mypy problemmatcher
- python3 root folder parameter
- raise error
- exit 1 io raise error
- only if python files are changed
- fix when no python files var not set
- pycodestyle 2.7.0
- make sure multiple files can be passed in python3-root-folder
- use extended grep
- use extended grep
- no escaping
- remove tr
- surround with quotes
- update black to 22.3.0
- Update requirements.txt
- black verbose
- add git fetch --all
- try to use micromamba instead of conda
